web/src/components/Inputs.js

```diff
@@ -9,10 +9,21 @@ class RichText extends Component {
     super(props);
 
     this.state = {
-      content: htmlToEditor(props.content)
+      content: htmlToEditor(props.content),
+      lastContent: props.content
     };
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if (nextProps.content !== this.state.lastContent) {
+      this.setState({
+        content: htmlToEditor(nextProps.content),
+        lastContent: nextProps.content
+      });
```

Comment:
Don't automatically return true here.  Doing so causes a non-stop render loop.  Instead, trigger a state update and wait for the state change before updating the component.